gc3apps/bf.uzh.ch/gcsvBulk.py

```diff
@@ -0,0 +1,216 @@
+#! /usr/bin/env python
+#
+#   gcsvBulk.py -- Front-end script for running a MATLAB function
```

Comment:
The name `gcsvBulk` is quite obscure to me; since the script runs a MATLAB function over multiple parameters, why not rename it to `gmatlab` or similar?